force-app/main/default/classes/RD2_ValidationService.cls

```diff
@@ -257,6 +258,29 @@ public with sharing class RD2_ValidationService {
         return errorRecords;
     }
 
+    private void validateCloseElevateReucrringDonation(npe03__Recurring_Donation__c rd, npe03__Recurring_Donation__c oldRd, ErrorRecord errorCollection) {
+        RD2_RecurringDonation recurringDonation = new RD2_RecurringDonation(rd);
+        if (!recurringDonation.isElevateRecord() || !recurringDonation.isClosed()) {
+            return;
+        }
+
+        Boolean isElevateFieldChange = rd.npe03__Contact__c != oldRd.npe03__Contact__c
```

Comment:
Can we extract this conditional into a separate method? 
`private Boolean isElevateFieldChanged(npe03__Recurring_Donation__c rd, npe03__Recurring_Donation__c oldRd)`